src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -1 +1,5 @@
 ; Please do not edit this file manually, it should only be updated through code fix application.
+### New Rules
```

Comment:
I know this is an auto-generated file, but the codefix was recently modified to respect markdownlint rules and add blank lines around headers. The analyzer already recognizes both forms.

```suggestion

### New Rules

```